src/main/java/com/exasol/adapter/document/UdfCallBuilder.java

```diff
@@ -83,31 +95,31 @@ private Select wrapStatementInStatementWithPostSelectionAndProjection(final List
      * statement.
      */
     private Select buildUdfCallStatement(final List<DocumentFetcher<DocumentVisitorType>> documentFetchers,
-            final RemoteTableQuery query, final String connectionName) throws IOException {
+            final RemoteTableQuery query) throws IOException {
         final Select doubleNestedSelect = StatementFactory.getInstance().select();
```

Comment:
Describing the function the SELECT serves instead of its structure would help make the code more readable.